### PR TITLE
.github: reduce headings one level

### PR DESCRIPTION
**Pull request by jmank88**
IMHO the PR template headings are loud and busy. This PR proposes reducing them one level, and removing 'Ticket'.
<hr/>
Old:
<hr/>

## Resolves Dependencies

<hr/>
New:
<hr/>

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Ticket
+### Ticket
```
Comment (jmank88) [inline .github/pull_request_template.md]: I'm wondering if this one even needs a heading at all, because there will almost always just be a single link.

**Review by chainchad — DISMISSED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (momentmaker) [reply .github/pull_request_template.md]: yeah maybe we just tell them to use one of those references in the comment?

Comment (momentmaker) [reply .github/pull_request_template.md]: @jmank88 should we just remove the ticket header altogether and just keep the comment instead?

Comment (jmank88) [reply .github/pull_request_template.md]:
```suggestion
```

**Review by momentmaker — APPROVED** [review]

**Review by erikburt — APPROVED** [review]

**Pull request merged**